### DISCUSSION TITLE
Different schemas with same models?

### DISCUSSION THREAD

**Discussion by maioradv** [prisma/prisma#22792] [Q&A]
### Question

Hi, it seems like is not possible to have different schemas but with same models.
Could be useful in order to create schemas with the same pattern of models.
schema_1 (users,post...)
schema_2 (users,post...)
in this scenario i can access the backend data changing schema depending on a customer (for example)

**Comment by ludralph** [comment]
Hi @maioradv 👋 

Thank you for raising this question.

Are you referring to multiple database schemas that is enabled by using the `multiSchema` preview feature? If yes, having each schema with the same model pattern is possible because this corresponds to having tables with the same name in different database schemas. You will need to map the table names to unique model names in your Prisma schema. This avoids name conflicts when you query models in Prisma Client. See this section of the [documentation](https://www.prisma.io/docs/orm/prisma-schema/data-model/multi-schema#tables-with-the-same-name-in-different-database-schemas).
```prisma
model BaseConfig {
  id Int @id

  @@map("config")
  @@schema("base")
}

model UserConfig {
  id Int @id

  @@map("config")
  @@schema("users")
}
```

**Reply by maioradv** [reply]
Yes, but this way force me to write the same kind of model of every schema, imagine if that config model is the same table for 10 schemas or more... that's not an effecient way to do that
a possibile solution could be @@schema("schema1,"schema2")
or schemas = [{
name: "base",
models:[Config,...]
},...]
for example

**Reply by ludralph** [reply]
Hi @maioradv, 

Can you please open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and document your thoughts so that our engineering team can take a look?

**Reply by janpio** [reply]
It seem you are looking for multi tenancy support "access the backend data changing schema depending on a customer (for example)". There are existing feature requests for that, please search for them and check if those would satisfy your use case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
